src/main/java/ch/epfl/biop/imaris/demo/MakeAndGetSpotsDemo.java

```diff
@@ -32,42 +32,42 @@ public static void main(String... args) throws Exception {
                     .setDiameter(1)
                     .setFilter("\"Quality\" above 15")
                     .isSubtractBackground(true)
-                    .setColor(new Integer[]{255,128,0})
+                    .setColor(new Integer[]{255, 128, 0})
                     .build()
                     .detect();
 
             // Adds the surface to the scene
             EasyXT.addToScene(detected_spots);
 
             // Gets an existing surface
-            ISpotsPrx got_spots = EasyXT.getSpots( "My Spots" );
+            ISpotsPrx got_spots = EasyXT.getSpots("My Spots");
 
             ISpotsPrx detected_ellipticSpots = SpotsDetector.Channel(0)
                     .setName("My Elliptic Region Grown Spots")
                     //.setDiameter( 1.0 )					// [Source Channel] Estimated XY Diameter
                     //.setAxialDiameter(2.0)				// [Source Channel] Estimated Z Diameter
-                    .setDiameterXYZ(  1.0 , 2.0 ) // alternative  that sets both parameters
-                    .isSubtractBackground(true)				// [Source Channel] Background Substraction = true
-                    .setFilter("\"Quality\" above 15.0")	// [Classify Spots] A String "\" as escape character
-                    .isRegionsFromLocalContrast(true) 		// [Spot Region Type] Region Growing  = Local Contrast (if set to false => intensity)
-                    .setRegionsThresholdManual(40)			// [Spot Regions] Region Growing Manual Threshold (Automatic Threshold is set to false if setting manual value)
-                    .isRegionsSpotsDiameterFromVolume(true)	// [Spot Regions] Region Growing Diameter = Diameter From Volume
-                    .isCreateRegionsChannel(false)			// [Spot Regions] Create Region Channel
-                    .setColor(new Integer[]{255,128,0})
+                    .setDiameterXYZ(1.0, 2.0) // alternative  that sets both parameters
+                    .isSubtractBackground(true)                // [Source Channel] Background Substraction = true
+                    .setFilter("\"Quality\" above 15.0")    // [Classify Spots] A String "\" as escape character
+                    .isRegionsFromLocalContrast(true)        // [Spot Region Type] Region Growing  = Local Contrast (if set to false => intensity)
+                    .setRegionsThresholdManual(40)            // [Spot Regions] Region Growing Manual Threshold (Automatic Threshold is set to false if setting manual value)
+                    .isRegionsSpotsDiameterFromVolume(true)    // [Spot Regions] Region Growing Diameter = Diameter From Volume
+                    .isCreateRegionsChannel(false)            // [Spot Regions] Create Region Channel
+                    .setColor(new Integer[]{255, 128, 0})
                     .build()
                     .detect();
 
             // Adds the surface to the scene
             EasyXT.addToScene(detected_ellipticSpots);
 
             // TODO : Display spots in Fiji
```

Comment:
This TODO is done, I think

Reply:
6638a0df1b6fa700d197995731ec5f1a34345a6d